helpers/HttpError.js

```diff
@@ -0,0 +1,7 @@
+const HttpError = (status, message) => {
```

Comment:
```suggestion
export const HttpError = (status, message) => {
``` можна відразу робити іменовий екмпорт, і не додавати ре-експорт в index